README.md

```diff
@@ -9,16 +9,21 @@ may well be used to fire requests to other services.
 
 
 * System dependencies
-    * none
+    * postgres 10.5  -> see setup below
 
 * Configuration
+    you may have to install postgres gem if you dont already have it 
+    
+    ```brew install libpq ```                                         # needed for the pg gem's native extensions
+   
+    ```PATH=$PATH:/usr/local/opt/libpq/bin```        # so that the pg gem can find the pq library
+    
+    ```gem install pg```
```

Comment:
`gem install pg` runs fine for me without the two new preceding libpq lines. It looks like we only need those if `gem install pg -- --with-pg-config /usr/local/opt/libpq/bin/pg_config` is run instead? If that's the approach we're taking then shouldn't we also update line 21 to reflect that?

Reply:
If we don't need libpq installed, then the way to test it is a fresh bundle install without libpq installed:

```bundle config path
mv $BUNDLE_PATH $BUNDLE_PATH.old   # get rid of existing bundle, to force a gem install
brew uninstall libpq
bundle install```

I tried this and it failed for me. I'd be interested to see what happens on your machines.